api/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
@mherman22 can you explain what the use of this is?

Reply:
So we need `mock-maker-inline` in order to run the module on java8 since mockito is version 3 where [tests fail without it](https://paste.ubuntu.com/p/pVZKhvNcjv/).

But without it on java17 and above, the tests will pass because on java17 we support mockito version 5 where it is not needed.

Take a look at https://github.com/openmrs/openmrs-core/blob/52294dedb8c3fb483d94cc1415dc6f360585cd35/pom.xml#L1176-L1258.

Ref: https://davidvlijmincx.com/posts/writing_higher_quality_tests_with_mockitos_inline_mock_maker/

Reply:
What I usually do in cases like this is add the Mockito Inline dependency to the project.

https://github.com/openmrs/openmrs-module-emrapi/blob/655416318f355ea64bec284ccd180de2efc2e328/pom.xml#L146-L151

I'm not sure what the best solution is here, so @dkayiwa, it's up to you to decide :)

Reply:
I think adding the dependency makes more sense. Since we have usecases of it in the eco anyway.

I will make the change